velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -268,6 +269,28 @@ TEST_F(ArithmeticTest, cosh) {
   EXPECT_TRUE(std::isnan(cosh(kNan).value_or(0)));
 }
 
+TEST_F(ArithmeticTest, rint) {
+  const auto rint = [&](std::optional<double> a) {
```

Comment:
This function can take double and return double. There is no caller that passes std::nullopt.

Reply:
Thanks, updated.